spec/models/miq_worker_spec.rb

```diff
@@ -198,11 +199,10 @@ def check_has_required_role(worker_role_names, expected_result)
     end
 
     it "uses passed in config" do
-      settings.store_path(:workers, :worker_base, :queue_worker_base, :ems_refresh_worker, :ems_refresh_worker_amazon, :memory_threshold, "1.terabyte")
-      stub_settings(settings)
-
-      config = VMDB::Config.new("vmdb")
-      actual = ManageIQ::Providers::Amazon::CloudManager::RefreshWorker.worker_settings(:config => config)[:memory_threshold]
+      settings.store_path(:workers, :worker_base, :queue_worker_base, :ems_refresh_worker,
+                          :ems_refresh_worker_amazon, :memory_threshold, "1.terabyte")
```

Comment:
it is kind of ok :). This test expecting to use parameters passed to method and not from ```::Settings```, I will make it more clear